scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -27,6 +27,9 @@ import kotlin.time.Duration.Companion.minutes
 import kotlin.time.Duration.Companion.seconds
 import kotlin.time.measureTimedValue
 
+import kotlinx.coroutines.Dispatchers
```

Comment:
Commit message: Make the last sentence a separate paragraph, and start capitalized after a colon ("issues: *T*his").